package.json

```diff
@@ -69,7 +70,7 @@
     "ts-node": "^10.2.0",
     "tsc-watch": "^4.2.9",
     "tslib": "^2.3.1",
-    "typescript": "~4.4.2",
+    "typescript": "^5.4.5",
```

Comment:
I'd suggest not to bump the typescript version, as it might break other implementations. We'll need a thorough code scan once we bump the version

Reply:
I bumped because `@farcaster/auth-client` library dependent to [viem](https://docs.farcaster.xyz/auth-kit/client/introduction#installation)
and viem has peerDependency to TypeScript >5.0.4 https://github.com/wevm/viem/blob/main/src/package.json#L117
So it breaks CI when I installed viem in `@magic-ext/farcaster`